parachain/test-parachains/adder/collator/src/main.rs

```diff
@@ -62,6 +62,7 @@ fn main() -> Result<()> {
 							config,
 							polkadot_service::IsCollator::Yes(collator.collator_key()),
 							None,
+							false,
```

Comment:
```suggestion
							true,
```

I assume we don't need that in tests